<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="show_geometry_button">
        <property name="text">
         <string>Geometry</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="show_element_values_button">
        <property name="text">
         <string>Element Values</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="kx_text">
        <property name="text">
         <string>Permeability in x-direction</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="d_text">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="h_text">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="w_text">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="show_nodal_values_button">
        <property name="text">
         <string>Nodal Values</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="t_text">
        <property name="text">
         <string>Thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="1" column="3" rowspan="2">
       <widget class="QPushButton" name="show_mesh_button">
        <property name="text">
         <string>Mesh</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ky_text">
        <property name="text">
         <string>Permeability in y-direction</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="element_size_label">
      <property name="text">
       <string>Max. el. size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="element_size_slider">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="report_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnew_action"/>
    <addaction name="actionopen_action"/>
    <addaction name="actionsave_action"/>
    <addaction name="actionaction"/>
    <addaction name="actionexit_action"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnew_action">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionopen_action">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave_action">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionaction">
   <property name="text">
    <string>save_as</string>
   </property>
  </action>
  <action name="actionexit_action">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
